python/ray/util/spark/databricks_hook.py

```diff
@@ -8,6 +8,20 @@
 
```

Comment:
We should make the hook pluggable and move this file to databricks internal codebase so that Ray repo has no DB specific code.

Reply:
agreed :). We can do it in a follow-up PR.